exercises/leap/canonical-data.json

```diff
@@ -1,37 +1,22 @@
 {
    "cases": [
       {
-         "description": "leap year in twentieth century",
-         "input": 1996,
-         "expected": true
-      },
-      {
-         "description": "odd standard year in twentieth century",
-         "input": 1997,
-         "expected": false
-      },
-      {
-         "description": "even standard year in twentieth century",
-         "input": 1998,
+         "description": "year not divisible by 4: not a leap year",
+         "input": 2015,
          "expected": false
       },
       {
-         "description": "standard year in nineteenth century",
-         "input": 1900,
-         "expected": false
+         "description": "year divisible by 4, not divisible by 100: leap year",
+         "input": 2016,
+         "expected": true
       },
       {
-         "description": "standard year in eighteenth century",
-         "input": 1800,
+         "description": "year divisible by 100, not divisible by 400: not a leap year",
```

Comment:
I disagree: every year divisible by 100 is divisible by 4 too, so to me the important part is that in this case rule 3 does not apply.